gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributeView.java

```diff
@@ -59,19 +62,19 @@ public void setTimes(FileTime lastModifiedTime, FileTime lastAccessTime, FileTim
   public boolean equals(@Nullable Object other) {
```

Comment:
Here an in other places. Find Bugs complains that we add `@Nullable` to an overridden method that is not defined as such.  "Method tightens nullness annotation on parameter", now clearly Object.equals does accept `null` and does not use `@Nullable` for obvious reasons, so this issue is harmless however I would like to avoid this warning.

Also, I am fine with using these annotation in specific contrib libraries but as an FYI, we decided not to use them
in the main gcloud-java library in order to avoid pulling, depending and exposing `com.google.code.findbugs` in our API.

I do wonder however if using the [checker-framework](http://types.cs.washington.edu/checker-framework/current/checker-framework-manual.html#maven) is not a better option as it seems to be
mentioned in the public Java [docs](https://blogs.oracle.com/java-platform-group/entry/java_8_s_new_type).

Thoughts? /cc @jart